e2e_tests/integration/viz.spec.ts

```diff
@@ -58,18 +60,96 @@ describe('Viz rendering', () => {
     cy.get(selectorNodeLabelAll, { timeout: 5000 }).should(
       'have.css',
       'background-color',
-      defaultColor
+      greyColor
     )
 
     // Change color and make sure color is changed
     cy.get(selectorNodeLabelAll, { timeout: 5000 }).click()
     cy.get('[data-testid="select-color-2"]').click()
-    cy.get('[data-testid="vizInspector"]') // Close grass editor
+    cy.get('[data-testid="cypherFrameSidebarVisualization"]').click() // Close grass editor
 
     cy.get(selectorNodeLabelAll, { timeout: 5000 }).should(
       'have.css',
       'background-color',
       orangeColor
     )
   })
+  it('can change default color of relationships', () => {
+    const selectorRelationshipsAll = `[data-testid="property-details-overview-relationship-type-*"]`
+
+    cy.executeCommand(':clear')
+    cy.executeCommand(':style reset')
+    cy.executeCommand(
+      'CREATE (a:TestLabel)-[:CONNECTS]->(b:TestLabel) RETURN a, b'
+    )
+    // Check that default color is set
+    cy.get(selectorRelationshipsAll, { timeout: 5000 }).should(
+      'have.css',
+      'background-color',
+      greyColor
+    )
+
+    // Change color and make sure color is changed
+    cy.get(selectorRelationshipsAll, { timeout: 5000 }).click()
+    cy.get('[data-testid="select-color-2"]').click()
+    cy.get('[data-testid="cypherFrameSidebarVisualization"]').click() // Close grass editor
```

Comment:
Maybe we can consolidate styles of `data-testid` to make all of them consistent in the future. All kebab-case or carmelCase.

Reply:
yes sounds good to agree on something in the team